internal/ethapi/testdata/eth_getBlockByHash-hash-1.json

```diff
@@ -4,17 +4,17 @@
   "extraData": "0x",
   "gasLimit": "0x47e7c4",
   "gasUsed": "0x5208",
-  "hash": "0x0da274b315de8e4d5bf8717218ec43540464ef36378cb896469bb731e1d3f3cb",
+  "hash": "0xeeb5c1852740ca4bbe65b0f57baf80634ed12a2b44affe30eec3fb54437c3926",
```

Comment:
What is the reason that all these json-files became different? Sorry for lazy question :)

Reply:
In the test backend I baked in an account for which we have the key in the genesis config so all the hashes changed :) I use this key to test transaction signing.